app/presenters/eastern_time_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class EasternTimePresenter
+  def initialize(timestamp)
+    @timestamp = timestamp
+  end
+
+  def to_s
+    I18n.t(
+      'event_types.timestamp',
+      date: eastern_timestamp.strftime('%B %e, %Y'),
```

Comment:
Sorry I meant capital `'%Z'` which is `EST` but yeah totally understood about full "Eastern"